src/psyclone/tests/psyir/transformations/transformations_test.py

```diff
@@ -35,6 +35,7 @@
 #         A. B. G. Chalk, STFC Daresbury Lab
 # Modified I. Kavcic, Met Office
 # Modified J. Henrichs, Bureau of Meteorology
+# Modified S. Valat, INRIA / LJK
```

Comment:
Maybe delete since at this point there isn't modifications at this specific file (but there is a couple of actually modified files in this PR where the contributors' names could be added).